chat-widget/src/common/utils.test.ts

```diff
@@ -119,7 +119,7 @@ describe("utils unit test", () => {
         expect(resultLTR).toEqual("ltr");
     });
 
-    it("Test changeLanguageCodeFormatForWebChat method", function() {
+    it("Test changeLanguageCodeFormatForWebChat method", function () {
 
```

Comment:
nit: remove extra line

Reply:
there is no extra line, that function space is taking right spacing.